# About me

Location: Saint Louis, Missouri, United States  
* [Email](sravani.valligari31@gmail.com)
* [LinkedIn](https://www.linkedin.com/in/sravani-valligari-545496170/)

## Education
### Master's in Bioinformatics and Computational Biology
**Saint Louis University** * Saint Louis, Missouri * 2024

### B. Tech, Bioinformatics
** Sathyabama Institute of Science and Technology** * Chennai, Tamil Nadu * 2021

## Technical Skills
* Languages: R programming, Python, C, C++, SQL
* Subject Expertise: Data Analysis and Visualization, Machine Learning Algorithms, Genetics, Drug Designing, Bioinformatic tools, NGS Data analysis, GWAS and Transcriptomic studies, Computational Biology and Systems Biology, Protein Structure Prediction.
* IDE & Tools: R Studio, VScode, Unix/Linux, Jupyter, Slack, GitHub, Tableau

## Professional Experience
### Bioinformatics Research Assistant | Donald Danforth Plant Science Center | St. Louis, MO
* Enhanced data analysis efficiency through optimizing an R Shiny application for ionomics data visualization.
* Boosted seed harvesting process and advanced data analysis on Arabidopsis and Setaria.
* Accelerated research project timelines using statistical and machine learning techniques for dataset analysis.

### Bioinformatics Research Intern | Averin Biotech Private Limited | India
* Improved protein-ligand interaction prediction accuracy by using bioinformatics tools like Accelrys Discovery Studio and MOE.
* Contributed to the identification of two pre-clinical therapeutic compounds through advanced molecular structure prediction.
* Enhanced drug discovery project, fostering cross-functional team collaboration.

## Projects

### R Shiny Web Application for FireMAGE Project
* Engineered a cutting-edge bioinformatics web application, significantly enhancing data visualization of genes, ionomics data, and traits, boosting user engagement and interpretability.
* Pioneered the integration of advanced visualization techniques, including violin plots, upset charts, and Venn diagrams, coupled with data tables for in-depth multi-trait and gene analysis, elevating the analytical capabilities for researchers.
Computational Systems Biology of Pityriasis lichenoides chronica (PLC) and its Comorbidities
* Conducted comprehensive gene expression and regulatory network analyses using PGP, Disgenet, and Disease Connect datasets, uncovering critical insights into PLC.
* Utilized Geo2R and Cytoscape to map out complex networks, identifying pivotal transcription factors and establishing novel pathway connections between PLC and Psoriasis, paving the way for groundbreaking drug target discoveries.

### Transcriptome Wide Association Studies for Alzheimer's Disease
* Developed an innovative framework combining GWAS and Transcriptomics to identify key single nucleotide variations linked to Alzheimer's, setting a new standard in genetic research.
* Exploited the Genome Asia 100k project to analyze SNP allele frequency across diverse populations, shedding unprecedented light on the genetic underpinnings of Alzheimer's disease, and contributing to global understanding of genetic diversity in disease predisposition.

## Publications
* Authored a research article titled "Computational Systems Biology of Pityriasis Lichenoides Chronica and Its Comorbidities," published by Inderscience Publishers, showcasing innovative findings in the field of computational biology and disease analysis.

[Link](https://doi.org/10.1504/IJBRA.2021.120206)